colossalai/utils/moe.py

```diff
@@ -0,0 +1,50 @@
+import torch.nn as nn
+import torch.distributed as dist
+from colossalai.core import global_context as gpc, moe_context as moe_env
+from colossalai.context import ParallelMode
+from .common import is_using_ddp
+
+
+def get_moe_param_dict(model: nn.Module):
+    """Returns a parameter dictionary, the key of which is the expert parallel
+    size of every parameter. Since the parameters in data parallelism is replicated
+    in each GPU, we set their ep_size to 1.
+
+    :param model: A pyTorch nn.model from which we get dict
+    :type model: torch.nn.Module
+    """
+    param_dict = dict()

```

Comment:
rename it to epsize_param_dict will be good for you readers.

Reply:
done